logstash-core/spec/logstash/filter_delegator_spec.rb

```diff
@@ -92,6 +93,7 @@ def filter(event)
       end
 
       it "increments the in/out of the metric" do
+        allow(metric).to receive(:increment).with(:time_ms, anything)
```

Comment:
Agree, with your explanation, but its either we have a dedicated test to make sure we record the `time_ms` in a filter execution or we rename the example to be more generic and mention `time_ms`, maybe renaming it like this `records filter executions details: in/out/time_ms` would do the trick?